packages/integration/config.d.ts

```diff
@@ -33,6 +33,24 @@ export interface Config {
        */
       token?: string;
     }>;
+    /** Integration configuration for Azure BlobStorage */
+    azureBlobStorage?: Array<{
+      /**
+       * derived from accountName, and defaults to accountName.blob.core.windows.net
```

Comment:
this could probably also have frontend visibility since it's derived from the account name